samples/meeting-av-filters/csharp/MeetingAVFilter/AppPackage/manifest.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "version": "1.0.0",
+  "id": "<<GUID ID>>",
+  "packageName": "com.microsoft.teams.videopp",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://microsoft.github.io/teams-videoapp-sample/",
+    "privacyUrl": "https://privacy.microsoft.com/en-us/privacystatement/",
+    "termsOfUseUrl": "https://www.microsoft.com/en-us/servicesagreement/"
+  },
+  "icons": {
+    "color": "icon-color.png",
+    "outline": "icon-outline.png"
+  },
+  "name": {
+    "short": "Teams Meeting AV Filter",
+    "full": "Teams Meeting AV Filter"
+  },
+  "description": {
+    "short": "This repo demonstrate a minimal Teams video app.",
+    "full": "This repo demonstrate a minimal Teams video app, clone this repo and change the video frame processing function"
```

Comment:
This sample demonstrate a minimal Teams video filter app....
Update this properly.